Tasks/DeployVisualStudioTestAgent/DownloadTestAgent.ps1

```diff
@@ -7,12 +7,12 @@ function ValidateSourceFile([string] $sourcePath)
 {
    if(! (Test-Path -Path $sourcePath))
    {
-        throw "Test agent source path '{0}' is not accessible to the test machine. Please check if the file exists and that test machine has access to that machine" -f $sourcePath
+        throw (Get-LocalizedString -Key "Test agent source path '{0}' is not accessible to the test machine. Please check if the file exists and that test machine has access to that machine" -f $sourcePath)
```

Comment:
check this cmd let Get-LocalizedString. Search for LocalizedString on VSO. even the cmd let wont be there on remote machine